cmd/admin_user_change_password.go

```diff
@@ -69,7 +73,9 @@ func runChangePassword(c *cli.Context) error {
 		return err
 	}
 
-	if err = user_model.UpdateUserCols(ctx, user, "passwd", "passwd_hash_algo", "salt"); err != nil {
+	user.MustChangePassword = c.Bool("must-change-password")
```

Comment:
We need to check `c.IsSet`?

Reply:
`c.Bool` defaults to `false` if it is not set which is what we want here.

Reply:
Then if `must-change-password` is not set, `must_change_password` column will always be updated to false even if previous data is true?

Reply:
As well, do we want to default it to true? That would match other change password flags we have in other commands.

Reply:
I'm also for default to true

Reply:
Because the CLI module has a weird boolean flag handling I changed the parameter to `must-not-change-password`. See https://github.com/urfave/cli/issues/740

Reply:
well you lookuped the wrong version!

BoolTFlag is not present in v2 and ondwards. just set the default value to true